### ISSUE TITLE
support node 12 ++

### ISSUE THREAD

**Issue by jinvillaz-zz** [senzil/cec-monitor#16]
Hello do you plan to support node 12 ++?
I'm using node 12 and doesn't work
environment:
OS: raspbian strech 
node: 12.13.1

in node 8.x works well

**Comment by pablodgonzalez** [comment]
Yes, of course, but right now we don't have time. I will leave these issue opened until we add the support

**Comment by jinvillaz-zz** [comment]
I need this feature, do you have a suggestion in my case? it's so complicated to modify your implementation in order to support node 12.

**Comment by pablodgonzalez** [comment]
Really I don't know, right now we don't have the CEC Adapters because those are in office an we all are working from home. Maybe if you add some stack trace, and more detailed issue.
We are planing check and update all our platform sbm.senzil.com to support node >=12 but the quarantine changed our plans. So, if you need it fast, I so sorry but it seems the quarantine does not finish for now.

**Comment by jinvillaz-zz** [comment]
Hello, In this moment I can't provide specific details about the issue because I don't have a TV with CEC. The next week I will have a TV with CEC and will test what is the issue, I need a solution because is a feature supported on node 8, but our software was updated to node 12 and this feature CEC was broken.
